WcaOnRails/config/locales/de.yml

```diff
@@ -4276,8 +4383,8 @@ de:
       title: Ablauf der Bewerbung
       #original_hash: 7b7de7e
       description_html: >-
-        Um die Anerkennung zu beantragen, fülle bitte <a
-        href='/regional-organizations/new'>dieses Formular</a> aus.
+        "Um die Anerkennung zu beantragen, fülle bitte <a
```

Comment:
Sorry for that, looks like I started the translation before the RO link PR (https://github.com/thewca/worldcubeassociation.org/pull/7042) was merged. Maybe better to close and resubmit.